x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/create_step/create_step.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, Fragment, useState } from 'react';
+import { EuiButton, EuiCheckbox, EuiFlexGroup, EuiFlexItem, EuiFormRow } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { CreateAnalyticsFormProps } from '../../../analytics_management/hooks/use_create_analytics_form';
+import { Messages } from '../../../analytics_management/components/create_analytics_form/messages';
+import { ANALYTICS_STEPS } from '../../page';
+import { BackToListPanel } from '../back_to_list_panel';
+
+interface Props extends CreateAnalyticsFormProps {
+  step: ANALYTICS_STEPS;
+}
+
+export const CreateStep: FC<Props> = ({ actions, state, step }) => {
+  const { createAnalyticsJob, startAnalyticsJob } = actions;
+  const {
+    isAdvancedEditorValidJson,
+    isJobCreated,
+    isJobStarted,
+    isModalButtonDisabled,
+    isValid,
+    requestMessages,
+  } = state;
+
+  const [checked, setChecked] = useState<boolean>(true);
+
+  if (step !== ANALYTICS_STEPS.CREATE) return null;
+
+  const handleCreation = async () => {
+    await createAnalyticsJob();
+
+    if (checked) {
+      startAnalyticsJob();
+    }
+  };
+
+  return (
+    <Fragment>
+      <Messages messages={requestMessages} />
```

Comment:
I wonder if it might be better putting any error messages below the Create and Start controls, as in the errors follow the action of creating the job?

![image](https://user-images.githubusercontent.com/7405507/83404216-25d7a200-a402-11ea-97ce-afcb60c542c8.png)

Reply:
Makes sense. Updated in 706f449459b3cc03f0bea7fe6483d423896e11b6